lib/tic_tac_toe.ex

```diff
@@ -1,12 +1,28 @@
 defmodule TicTacToe do
   alias TicTacToe.Display
+  alias TicTacToe.Console
   alias TicTacToe.Presenter
+  alias TicTacToe.Player
+  alias TicTacToe.HumanPlayer
+  alias TicTacToe.Board
 
   def run(opts \\ []) do
     board = Keyword.get(opts, :board, List.duplicate(nil, 9))
-    display = Keyword.get(opts, :display, Display)
+    display = Keyword.get(opts, :display, %Console{})
     presenter = Keyword.get(opts, :presenter, Presenter)
+    player = Keyword.get(opts, :player, %HumanPlayer{token: "X"})
```

Comment:
Coming up, we might want to look at removing some of these defaults. Defaults can be really great, but they can also cause a headache. One option here is you could define a struct that should include keys for all of these and glue them together. That would also push the specific configuration into the `main` (i.e. mix task)

No changes needed now, just something to consider in later on.

Reply:
Noted